tools/scripts/upload_alerts_to_aws.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+import boto3
+import json
+import argparse
+import gzip
+import io
+import json
+import os
+import xml.etree.ElementTree as ET
+from typing import Any, Dict, List
+
+import boto3  # type: ignore[import]
+import rockset  # type: ignore[import]
+
+S3_RESOURCE = boto3.resource("s3")
+def upload_to_s3(
+    bucket_name: str,
+    key: str,
+    docs: List[Dict[str, Any]],
+) -> None:
+    print(f"Writing {len(docs)} documents to S3")
+    body = io.StringIO()
+    for doc in docs:
+        json.dump(doc, body)
+        body.write("\n")
+
+    S3_RESOURCE.Object(
+        f"{bucket_name}",
+        f"{key}",
+    ).put(
+        Body=gzip.compress(body.getvalue().encode()),
+        ContentEncoding="gzip",
+        ContentType="application/json",
+    )
+    print("Done!")
+
+def get_recent_alerts(orgname, reponame):
+    rockset_api_key = os.environ["ROCKSET_API_KEY"]
+    rockset_api_server = "api.rs2.usw2.rockset.com"
+    rs = rockset.RocksetClient(
+        host="api.usw2a1.rockset.com", api_key=os.environ["ROCKSET_API_KEY"]
+    )
+
+    # Define the name of the Rockset collection and lambda function
+    collection_name = "commons"
+    lambda_function_name = "get_relevant_alerts"
+    query_parameters = [
+        rockset.models.QueryParameter(name="repo", type="string", value=reponame),
+        rockset.models.QueryParameter(name="organization", type="string", value=orgname),
+    ]
+    api_response = rs.QueryLambdas.execute_query_lambda(
+        query_lambda=lambda_function_name,
+        version="ba8b0b824e799c45",
```

Comment:
You could read from `torchci/rockset/prodVersions.json` to get the version there to avoid copy paste mistake.  The same way is done here https://github.com/pytorch/test-infra/blob/main/torchci/scripts/queue_alert.py#L141